src/lava/lib/optimization/solvers/generic/builder.py

```diff
@@ -245,7 +246,8 @@ def constructor(self, proc):
             )
             macrostate_reader.read_gate_solution_out.connect(
                 macrostate_reader.solution_readout_solution_in)
-
+            macrostate_reader.solution_readout.acknowledgemet.connect(

```

Comment:
This should be 

macrostate_reader.solution_readout.acknowledgeme**n**t.connect(